### ISSUE TITLE
Block Group or Tract?

### ISSUE THREAD

**Issue by gregmacfarlane** [gregmacfarlane/parks_access#8]
Related to #5 is the question of the spatial aggregation of the model. The 500 Cities data and the Census sociodemographic information is *only* available at the tract level. But we could easily enough calculate the accessibility at the block-group level and simply project the tract-level data down to that level.

**Comment by gregmacfarlane** [comment]
Given the findings of #5, we will only look at this using Euclidean distances. We run into an issue
pretty earlier, however, in trying to simply calculate the models we've been doing at the tract level. Specifically, the repeated observations appear to create collinearity problem that prevents the numerical model estimation procedures. The collinearity problem appears to specifically be caused by the lagged $X$ variables in the function. This makes sense, because at the block group level the surrounding block groups all have precisely the same lagged value.

**Comment by gregmacfarlane** [comment]
The models below show SEM estimates of physical activity as a function of euclidean access
and the other predictor variables. While some of the variables have the same sign, scale, and interpretation across models, some --- specifically college and young adults --- do not. It is impossible to tell whether this is a result of the block group-level accessibility, or a manifestation of the ecological inference fallacy, or something else. 

```
====================================================
                     Tracts          BG             
----------------------------------------------------
(Intercept)              5.6200 *         9.0269 ***
                        (2.7406)         (1.5723)   
log(density)             0.1740 *         0.0505    
                        (0.0691)         (0.0402)   
log(income)              5.8583 ***       5.6940 ***
                        (0.2095)         (0.1207)   
fulltime                 0.1420 ***       0.1369 ***
                        (0.0083)         (0.0047)   
college                  0.0116           0.0283 ***
                        (0.0111)         (0.0062)   
single                  -0.0331 ***      -0.0305 ***
                        (0.0074)         (0.0042)   
youth                   -0.1309 ***      -0.1250 ***
                        (0.0124)         (0.0070)   
young_adults             0.0223 *         0.0156 ** 
                        (0.0097)         (0.0054)   
seniors                  0.0431 ***       0.0389 ***
                        (0.0125)         (0.0071)   
black                   -0.0517 ***      -0.0562 ***
                        (0.0042)         (0.0026)   
asian                   -0.0773 ***      -0.0767 ***
                        (0.0053)         (0.0031)   
hispanic                -0.1026 ***      -0.1073 ***
                        (0.0046)         (0.0027)   
other                   -0.0297          -0.0336    
                        (0.0390)         (0.0224)   
euc_access               0.1334           0.0014    
                        (0.0711)         (0.0394)   
lambda                   0.7890 ***       0.9052 ***
                        (0.0157)         (0.0058)   
----------------------------------------------------
Num. obs.             2102             6402         
Parameters              16               16         
Log Likelihood       -4498.0118      -11966.8091    
AIC (Linear model)   10080.5829       30278.3483    
AIC (Spatial model)   9028.0237       23965.6183    
LR test: statistic    1054.5592        6314.7301    
LR test: p-value         0.0000           0.0000    
====================================================
*** p < 0.001, ** p < 0.01, * p < 0.05

```

It should be noted that in this SEM specification, the accessibility term of the tract model is not significant though it appears of the same sign and magnitude as in the SDEM model shown in the manuscript. We would argue the appropriate econometric model for this problem is the SDEM based on the discussion in the manuscript. There is no guarantee that an SDEM at the block group level (could it be estimated) would show a negligible effect akin to the `BG` model coefficient shown here.

**Comment by gregmacfarlane** [comment]
tldr; We actually can't estimate blockgroup-level SDEM models. The comparison of an SEM at the tract and blockgroup levels suggest that projecting all tract information down to the block group is not appropriate.
